docs/trulens/tracking/instrumentation/llama_index.ipynb

```diff
@@ -0,0 +1,497 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 📓 🦙 LlamaIndex Integration\n",
+    "\n",
+    "TruLens provides TruLlama, a deep integration with LlamaIndex to allow you to\n",
+    "inspect and evaluate the internals of your application built using LlamaIndex.\n",
+    "This is done through the instrumentation of key LlamaIndex classes and methods.\n",
+    "To see all classes and methods instrumented, see *Appendix: LlamaIndex\n",
+    "Instrumented Classes and Methods*.\n",
+    "\n",
+    "In addition to the default instrumentation, TruChain exposes the\n",
+    "*select_context* and *select_source_nodes* methods for evaluations that require\n",
+    "access to retrieved context or source nodes. Exposing these methods bypasses the\n",
+    "need to know the json structure of your app ahead of time, and makes your\n",
+    "evaluations re-usable across different apps.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example usage\n",
+    "\n",
+    "Below is a quick example of usage. First, we'll create a standard LlamaIndex query engine from Paul Graham's Essay, *What I Worked On* "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from llama_index.core import VectorStoreIndex\n",
+    "from llama_index.readers.web import SimpleWebPageReader\n",
+    "\n",
+    "documents = SimpleWebPageReader(html_to_text=True).load_data(\n",
+    "    [\"http://paulgraham.com/worked.html\"]\n",
+    ")\n",
+    "index = VectorStoreIndex.from_documents(documents)\n",
+    "\n",
+    "query_engine = index.as_query_engine()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To instrument an LlamaIndex query engine, all that's required is to wrap it using TruLlama."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
```

Comment:
Do docs need notebook outputs? We could save a lot of space in our git history by stripping them. I have a pre-commit hook to do so but can exclude the docs directory if needed

Reply:
some are useful to have. would be good to exlude the docs directory from that hook